contracts/ServiceNodeContribution.sol

```diff
@@ -86,6 +91,11 @@ contract ServiceNodeContribution is Shared {
         operator = tx.origin; // NOTE: Creation is delegated by operator through factory
         blsPubkey = _blsPubkey;
         serviceNodeParams = _serviceNodeParams;
+
+        for (uint256 i = 0; i < _reservedContributors.length; i++) {
+            reservedContributorAddresses.push(_reservedContributors[i].addr);
+            reservedContributions[_reservedContributors[i].addr] += _reservedContributors[i].stakedAmount;
```

Comment:
Needs a check that this won't exceed the staking requirement.  (But also see comment about operator contribution not being known yet).